Diff/Diff.sublime-syntax

```diff
@@ -307,10 +307,10 @@ contexts:
 ###############################################################################
 
 variables:
-  conflict_begin: <{5,}
-  conflict_end: '>{5,}'
-  conflict_split: ={5,}
-  conflict_base: \|{5,}
+  conflict_begin: <{7}
+  conflict_end: '>{7}'
+  conflict_split: ={7}
+  conflict_base: \|{7}
```

Comment:
Is it feasible to do `conflict_marker_size: '{7}'` and then `conflict_begin: <{{conflict_marker_size}}` etc.?

Sorry I'm AFK and cannot test. I would approve this as-is, but wanted to suggest.

Reply:
Applied.